pkg/crinit/crinit.go

```diff
@@ -28,21 +29,15 @@ import (
 
 // NewClusterregistryCommand creates the `clusterregistry` command.
 func NewClusterregistryCommand(out io.Writer, defaultServerImage, defaultEtcdImage string) *cobra.Command {
-	cmds := &cobra.Command{
+	rootCmd := &cobra.Command{
 		Use:   "crinit",
 		Short: "crinit runs a cluster registry in a Kubernetes cluster",
 		Long:  "crinit bootstraps and runs a cluster registry as a Deployment in an existing Kubernetes cluster.",
-		Run:   runHelp,
 	}
 
-	// From this point and forward we get warnings on flags that contain "_" separators
-	cmds.SetGlobalNormalizationFunc(flag.WarnWordSepNormalizeFunc)
+	rootCmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
+	rootCmd.SetGlobalNormalizationFunc(apiserverflag.WordSepNormalizeFunc)
```

Comment:
Why not use the warning method any longer?

Reply:
It looks like there are some flags that `glog` adds that have underscores, and this causes a warning to be printed out that there's not much we can do about.

That said, the other k8s tools don't run into this, so there's probably a better way around it.

Reply:
The way `kubefed` handles it - or rather how it's handled in the `kubectl` Factory API that `kubefed` relies on - is that it sets the non-warn `WordSepNormalizeFunc` function first, then adds the Go flag set to pflags, then sets the warn `WarnWordSepNormalizeFunc` function. I've seen others just set the non-warn function. I think I prefer to see the warnings and just suppress them for the glog flags.